lib/hooks/userspace/hook-mongodb-core.js

```diff
@@ -28,7 +28,7 @@ function nextWrap(next) {
   return function next_trace(cb) {
     var root = cls.getRootContext();
     if (!root) {
-      agent.logger_.warn('Cannot create mongo span outside of a supported framework.');
+      agent.logger.warn('Cannot create mongo span outside of a supported framework.');
```

Comment:
Reword similarly to http above. Users don't know what a span is. Tell them the implication instead.